sql/testdata/insert

```diff
@@ -19,7 +19,7 @@ statement error missing "k" primary key column
 INSERT INTO kv (v) VALUES ('a')
 
 statement ok
-INSERT INTO kv VALUES ('nil', NULL)
+INSERT INTO kv (k) VALUES ('nil')
```

Comment:
note I couldn't simply add this test case because of https://github.com/cockroachdb/cockroach/issues/1909